source/includes/fundamentals/code-snippets/bson-extra-elements/BsonExtraElementsExample.java

```diff
@@ -0,0 +1,70 @@
+package fundamentals;
+
+import static com.mongodb.MongoClientSettings.getDefaultCodecRegistry;
+import static org.bson.codecs.configuration.CodecRegistries.fromProviders;
+import static org.bson.codecs.configuration.CodecRegistries.fromRegistries;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.bson.Document;
+import org.bson.codecs.configuration.CodecRegistry;
+import org.bson.codecs.pojo.PojoCodecProvider;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.Updates;
+import com.mongodb.client.result.UpdateResult;
+
+public class BsonExtraElementsExample {
+
+	private static final String DOC_NAME_VALUE = "MDB0123";
+
+	private static CodecRegistry getCodecRegistry() {
+		PojoCodecProvider pojoCodecProvider = PojoCodecProvider.builder().automatic(true).build();
+		return fromRegistries(getDefaultCodecRegistry(), fromProviders(pojoCodecProvider));
+
```

Comment:
Nit: extra whitespace

Reply:
> The indentation is still inconsistent.

Sorry, strange conversion of tabs and spaces makes it look ok in the IDE. Need to adjust my IDE settings as mentioned in https://github.com/mongodb/docs-java/pull/242#discussion_r940305061